snapshot/policy/retention_policy.go

```diff
@@ -160,6 +168,14 @@ func (r *RetentionPolicy) getRetentionReasons(i int, s *snapshot.Manifest, cutof
 			continue
 		}
 
+		if *r.KeepMinDays != *newOptionalInt(0) {
+			if !s.StartTime.Before(cases[6].cutoffTime) {
+				keepReasons = []string{"Inside max retention days"}
```

Comment:
Should this say "min"?
```suggestion
				keepReasons = []string{"Inside min retention days"}
```

Reply:
Yes, that should be min